lib/utils/provider.ts

```diff
@@ -40,7 +40,17 @@ export const provider = (settings?: Partial<Settings>): Provider => ({
             }
         }
 
-        return send(settings.provider || '', command)
+        let abortSignal = undefined
```

Comment:
Please move this logic inside `send()` and pass to it the `timeout`. This way `send()` accepts `settings` only, and `batchedSend()` will automatically grab the timeout logic as well.